src/app/clusters/fixed-label-server/fixed-label-cluster.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ *    Copyright (c) 2025 Project CHIP Authors
```

Comment:
you kept a lot of code from the old cluster I presume, so change this to 2021-2025

```suggestion
 *    Copyright (c) 2021-2025 Project CHIP Authors
```